src/endpoints/workers/mod.rs

```diff
@@ -0,0 +1,39 @@
+use crate::framework::response::ApiResult;
+
+use serde::Deserialize;
+
+mod create_route;
+mod delete_route;
+mod list_routes;
+
+pub use create_route::{CreateRoute, CreateRouteParams};
+pub use delete_route::DeleteRoute;
+pub use list_routes::ListRoutes;
+
+/// Workers KV Route
+/// Routes are basic patterns used to enable or disable workers that match requests.
+/// https://api.cloudflare.com/#worker-routes-properties
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
+pub struct WorkersRoute {
+    /// Namespace identifier tag.
+    pub id: String,
+    /// The basic pattern that should map to the script
+    pub pattern: String,
```

Comment:
Is there any interest in combining this struct and `WorkersRouteIdOnly` and using `#[serde_with::skip_serializing_none]` to only try to serialize `pattern` and `script` if those fields are present in the API response?

Reply:
so the reason i did this is that one endpoint only returns an id, and others return all of them; they are not "optional" exactly, there are just different structures based on the endpoint. if we do skip, i would say we should make a PR to the config api to return the whole object all the time instead of changing the entity that is returned based on the endpoint.

Reply:
merging this way; if the api changes it won't break the client, but we'll just miss out on those fields until we update.